src/macaron/provenance/provenance_finder.py

```diff
@@ -476,56 +380,60 @@ def find_provenance_from_ci(analyze_ctx: AnalyzeContext, git_obj: Git | None) ->
     return None
 
 
-def download_provenances_from_github_actions_ci_service(ci_info: CIInfo) -> None:
+def download_provenances_from_ci_service(ci_info: CIInfo, download_path: str) -> None:
```

Comment:
Hmm if we rename this function to `from_ci_service`, does it mean that we can use this for all supported ci services, and not just only GitHub Actions?.

Reply:
That should be possible.